Casks/f/fontcreator.rb

```diff
@@ -0,0 +1,23 @@
+cask "fontcreator" do
+  version "15.0.0.3014"
+  sha256 :no_check
```

Comment:
As per the versioned url, 
```suggestion
  sha256 "2cdeb67b065d006f7713c5b4357df42bd0e05a59e511aca6d231e827c120a267"
```